client/src/ui/organisms/placement/index.scss

```diff
@@ -19,6 +25,7 @@
       padding: 0.825rem;
 
       img {
+        margin: 0;
```

Comment:
Did it previously have a margin? Can you provide before/after screenshots in the PR description?